### PR TITLE
feat: allow custom package configuration file path

### PR DESCRIPTION
**Pull request by tpetillot**
Ref: HYB-596

### CODE REVIEW

**Review by slandelle — APPROVED** [review]

File: src/main/java/io/gatling/mojo/EnterpriseDeployMojo.java [hunk 1]
```diff
@@ -24,18 +24,21 @@
...
 import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
 import org.apache.maven.plugins.annotations.ResolutionScope;
 
 @Execute(goal = "enterprisePackage")
 @Mojo(name = "enterpriseDeploy", requiresDependencyResolution = ResolutionScope.TEST)
 public final class EnterpriseDeployMojo extends AbstractEnterprisePluginMojo {
   public static final String CONTEXT_ENTERPRISE_DEPLOY_INFO = "enterprise_deploy_info";
 
+  @Parameter(property = "gatling.enterprise.packageConf")
```
Comment (Obero) [inline src/main/java/io/gatling/mojo/EnterpriseDeployMojo.java:35]: Choose wisely :older_adult:

**Review by Obero — APPROVED** [review]

**Review by Obero — CHANGES_REQUESTED** [review]
Need to bump commons version

File: src/main/java/io/gatling/mojo/EnterpriseDeployMojo.java (hunk 1 above)
Comment (Obero) [reply src/main/java/io/gatling/mojo/EnterpriseDeployMojo.java:35]: packageDescriptorFilename

**Review by Obero — APPROVED** [review]
Upgrade done, including this PR content: https://github.com/gatling/gatling-maven-plugin/pull/168#pullrequestreview-2008184449

**Pull request merged**